src/verification/ScratchUserCheck.php

```diff
@@ -8,7 +8,9 @@ class ScratchUserCheck {
 
 	private static function fetchProfile($username, &$isScratcher, &$joinedAt, &$error) {
 		$url = sprintf(self::PROFILE_URL, $username);
-		$html = @file_get_contents($url);
+		$html = @file_get_contents($url, false, stream_context_create(
```

Comment:
![image](https://github.com/InternationalScratchWiki/scratch-confirmaccount-v3/assets/1960707/eba1224f-bd49-43c6-ad6c-1384438478c3)
Make sure to wrap the outer part of the array properly. I believe it should be:
```php
stream_context_create([
  "http" => ["header" => "Accept-Language: en\r\nCookie: scratchlanguage=en"]
])```

Also a more minor detail: single quotes are preferred over double quotes unless you are specifically intending to use variable interpolation or escape sequences.